code/game/objects/items/control_wand.dm

```diff
@@ -224,15 +261,30 @@
 		to_chat(user, "<span class='danger'>The [D] airlock has no power!</span>")
 		return
 
-	if(D.check_access(ID))
-		D.add_hiddenprint(user)
-		if(D.density)
-			D.open()
-		else
-			to_chat(user, "<span class='danger'>The [D] airlock is already open!</span>")
+	if(!D.check_access(ID))
+		to_chat(user, "<span class='danger'>[src] does not seem to have a key for the [D] airlock's access panel!</span>")
+		return
+
+	D.add_hiddenprint(user)
+	if(D.density)
+		D.open()
+	else
+		to_chat(user, "<span class='danger'>The [D] airlock is already open!</span>")
 
+/obj/item/door_remote/janikeyring/access_windoor(obj/machinery/door/window/D, mob/user)
+	if(!(D.has_power()))
+		to_chat(user, "<span class='danger'>[D] has no power!</span>")
+		return
+
+	if(!D.check_access(src.ID))
+		to_chat(user, "<span class='danger'>[src] does not seem to have a key for the [D]'s access panel!</span>")
+		return
+
+	D.add_hiddenprint(user)
+	if(D.density)
+		D.open()
 	else
-		to_chat(user, "<span class='danger'>[src] does not seem to have a key for the [D] airlock's access panel!</span>")
+		to_chat(user, "<span class='danger'>The [D] is already open!</span>")
```

Comment:
I don't know if `The` is needed here

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/61080616/5e2f5d27-478c-49d9-81dc-2dfd3eb10659)
is it weird?

Reply:
I remember in the past we removed a lot of instances of `the` b/c string literals would automatically tack on the adjective for some stuff. But frankly it doesn't matter that much.